### ISSUE TITLE
Counter `x of y` is wrong on slick dots button's `aria-label`

### ISSUE THREAD

**Issue by bboro** [kenwheeler/slick#4195]
The counter for the `aria-label` for slick dots is calculating the totals wrong when `slidesToShow` and `slidesToScroll` settings are different. Currently slick's dividing the total number of slides by `slidesToShow`. This ends up creating labels for slick dots such as `aria-label="6 of 2"` which is plain wrong. It should instead divide the total number of slides by `slidesToScroll`. 

====================================================================

###### `[http://jsfiddle.net/3dau6hkb/3/]`  

====================================================================

#### Steps to reproduce the problem

(Same as jsfiddle link above)

1. Create a slick slideshow with 6 slides.
2. Set `slidesToShow: 3`, `slidesToScroll: 1`, `dots: true`
3. Inspect the `aria-label` property of the dots buttons. 

====================================================================

#### What is the expected behaviour?

_Dots buttons_* should say `aria-label="[x] of 6"`, the total number of dots should be correctly marked as 6.

====================================================================

#### What is observed behaviour?

_Dots buttons_* say `aria-label="[x] of 2"`, the total number of dots is calculated as 2 instead of 6.

====================================================================

#### More Details

- Which browsers/versions does it happen on?
  - Not applicable
- Which jQuery/Slick version are you using?
  - Latest from CDN for jQuery and Slick
- Did this work before?
  - Unsure if this is a regression.

**Comment by ahmadalfy** [comment]
That's a nice catch, thank you.

**Comment by ahmadalfy** [comment]
Merged
